source/plugins/go/src/extension/extension.go

```diff
@@ -23,7 +22,7 @@ var containerType string
 
 func GetInstance(flbLogger *log.Logger, containertype string) *Extension {
 	once.Do(func() {
-		singleton = &Extension{make(map[string]string)}
+		singleton = &Extension{make(map[string]string), make(map[string]string)}
 		flbLogger.Println("Extension Instance created")

```

Comment:
why do we need 2 maps, cant we use the single map here ?

Reply:
As discussed offline with Ganga decided to keep two maps